wordpress/mu-plugins/skyyrose-anon-cache-guard.php

```
@@ -0,0 +1,118 @@
+			 */
+			class SkyyRose_Cacheable_Session_Handler extends WC_Session_Handler {
+				/**
+				 * Set the session cookie unless this view must stay cacheable.
+				 *
+				 * @param bool $set Whether the cookie should be set.
+				 */
+				public function set_customer_session_cookie( $set ) {
+					if ( $set && skyyrose_is_cacheable_anon_pageview() ) {
+						return;
+					}
+					parent::set_customer_session_cookie( $set );
```

PHPStan (2.2.2)

[error] SkyyRose_Cacheable_Session_Handler::set_customer_session_cookie() calls parent::set_customer_session_cookie() but SkyyRose_Cacheable_Session_Handler does not extend any class.


(class.noParent)